test/Mvc6Framework45.FunctionalTests/Models/AccountViewModels.cs

```diff
@@ -4,6 +4,7 @@
 using System.Threading.Tasks;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.Rendering;
```

Comment:
I'm not an expert on those TFMs, but my understanding is that you should pretend like `netstandardapp` never existed, and use `netcoreapp` instead, though they aren't *exactly* equivalent. This is where @davidfowl could likely help more.